### ISSUE TITLE
[Other - Accessibility] Partly truncated text at high zoom. (00.00.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#72160]
#### VFS team name
Veteran Facing Forms

#### VFS product name
20-10206

#### VFS feature name
FOIA / PA request

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
On [/disability-exam-details](https://staging.va.gov/records/request-personal-records-form-20-10206/disability-exam-details) at 400% zoom, the "Yes" button's text is cut off. THe majority of the text is visible so most users should be able to complete the task, but it's a degraded experience.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
This was already [flagged by Jeana](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/856), and there's some discussion of options in the comments of the ticket. I think my preference would be to update to the v3 modal in the forms system array code, but I defer to you all to decide what gives you the best result with the appropriate risk trade-off.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by tbaker1026** [comment]
If we change this, it might change for other forms that use it.  This would impact a lot forms, need to coordinate this, and get right people to approve PR.

**Comment by jeana-adhoc** [comment]
This has been resolved.

https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/856
